dash/webapp/pages/pa11y.tsx

```diff
@@ -35,7 +36,7 @@ const Issue = styled.div`
 `;
 
 const Index: FC = () => {
-  const [resultsList, setResultsList] = useState(null);
+  const [resultsList, setResultsList] = useState([]);
```

Comment:
This was giving `Property 'results' does not exist on type 'never'.` on line 62, so I changed how we treat it a bit - open to suggestions!